tests/x11regressions/gnote/gnote_search_all.pm

```diff
@@ -21,6 +21,8 @@ sub run {
     x11_start_program("gnote");
     assert_screen "gnote-first-launched", 10;
     send_key "ctrl-f";
+    # The gnote interface is slow. So we can't start immediatly searching. We need to wait
+    sleep(2);
```

Comment:
You're right. That function is better suitable for here.
I'm going to change it